### PR TITLE
NOISSUE: fix launchnet start

### PR DESCRIPTION
**Pull request by sexyhamster**
Functests still do not work, but at least network starts

### CODE REVIEW

File: ledger-core/v2/application/testutils/launchnet/launchnet.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func launchnetPath(a ...string) (string, error) {
 	cwdList := strings.Split(cwd, "/")
 	var count int
 	for i := len(cwdList); i >= 0; i-- {
-		if cwdList[i-1] == "insolar" && cwdList[i-2] == "insolar" {
+		if cwdList[i-1] == "v2" && cwdList[i-2] == "ledger-core" {
```
Comment (afiskon) [inline ledger-core/v2/application/testutils/launchnet/launchnet.go]: this line can access cwdList[-1] and cwdList[-2]. I believe for condition should be chaned.

File: ledger-core/v2/application/testutils/launchnet/launchnet.go [hunk 2]
```diff
@@ -415,7 +415,7 @@ func startNet() error {
 		_ = os.Chdir(cwd)
 	}()
 
-	for cwd[len(cwd)-15:] != "insolar/assured-ledger/ledger-core/v2/" {
+	for cwd[len(cwd)-37:] != "insolar/assured-ledger/ledger-core/v2" {
```
Comment (afiskon) [inline ledger-core/v2/application/testutils/launchnet/launchnet.go]: if len(cwd) < 37 this code will panic.

File: ledger-core/v2/application/testutils/launchnet/launchnet.go (hunk 1 above)
Comment (sexyhamster) [reply ledger-core/v2/application/testutils/launchnet/launchnet.go]: yep

File: ledger-core/v2/application/testutils/launchnet/launchnet.go (hunk 2 above)
Comment (sexyhamster) [reply ledger-core/v2/application/testutils/launchnet/launchnet.go]: yep

**Pull request merged**